### PR TITLE
docs: add mock heroes to the code review tabs

### PR DESCRIPTION
**Pull request by erikologic**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by mrenrich84** [comment]
I signed it!

File: aio/content/tutorial/toh-pt2.md [hunk 1]
```diff
@@ -237,6 +237,10 @@ Your app should look like this <live-example></live-example>.
 Here are the code files discussed on this page, including the `HeroesComponent` styles.
 
 <code-tabs>
+  <code-pane header="src/app/mock-heroes.ts" path="toh-pt2/src/app/mock-heroes.ts">
```
Comment (brandonroberts) [inline aio/content/tutorial/toh-pt2.md]: Put a single line break before and after the `code-pane`

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Remove the `(docs-infra)` scope from the git commit message

**Comment by brandonroberts** [comment]
@mrenrich84 this [git commit message](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#subject) should in present tense. Update to `docs: add mock heroes to the code review tabs for TOH pt. 2`

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Caretaker note: integration_test failure is unrelated

**Pull request closed without merging**